<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tetrisClass</class>
 <widget class="QMainWindow" name="tetrisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>tetris</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tetris.qrc">
    <normaloff>:/tetris/x64/Debug/avatar.png</normaloff>:/tetris/x64/Debug/avatar.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>181</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Счет</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLCDNumber" name="score"/>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>191</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>следующий элемент</string>
     </property>
     <widget class="QPushButton" name="games">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>190</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Начать игру</string>
      </property>
     </widget>
     <widget class="QPushButton" name="about">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>О программе</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="gamepoll">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>431</width>
      <height>601</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Рекорды</string>
    </property>
    <widget class="QTableWidget" name="table">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>181</width>
       <height>241</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>82</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>82</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>43</number>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Игрок</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Счет</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="Name">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>player</string>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Как прикажете вас звать?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>191</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="pic2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>181</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>181</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="pic1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>181</width>
       <height>231</height>
      </rect>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="change">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>сменить фон</string>
     </property>
    </widget>
    <widget class="QPushButton" name="music">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tetris.qrc"/>
 </resources>
 <connections/>
</ui>
